src/scripts/release.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+USAGE="Usage: release.sh <prepare|deploy> MAJOR.MINOR.PATCH"
+COMMAND=$1
+VERSION=$2
+
+function fatal {
+    echo "$@" >&2
+    exit 1
+}
+if [ "$#" -ne 2 ]; then
+    fatal "$USAGE"
+fi
+if ! [[ $VERSION =~ ^[[:digit:]+\\.[:digit:]+\\.[:digit:]]+$ ]]; then
+	fatal "$USAGE"
+fi
+if [[ $COMMAND != "prepare" && $COMMAND != "deploy" ]]; then
+	fatal "$USAGE"
+fi
+if [[ $(git status --porcelain 2> /dev/null | tail -n1) != "" ]]; then
```

Comment:
Sorry, didn't pick this up earlier. Either use `-n` here or `= ""` instead of `-z` below.

Reply:
Done (changed the bottom one).